src/main/org/nlogo/compiler/CompilerMain.scala

```diff
@@ -29,7 +29,7 @@ private object CompilerMain {
     for(procedure <- structureResults.procedures.values.asScala) {
       procedure.topLevel = subprogram
       val tokens =
-        new IdentifierParser(program, oldProcedures, structureResults.procedures, false)
+        new IdentifierParser(program, oldProcedures, structureResults.procedures, forgiving = true)
```

Comment:
Good call.

Here's one place where it was still false: https://github.com/NetLogo/NetLogo/blob/cd253418c13860c1ee1c5874be66480fc2019982/src/main/org/nlogo/compiler/CompilerMain.scala#L32

Pretty sure this is incorrect though, and would've prevented the use of symbols in reporters given at the command line.